### PR TITLE
ROC-2290 Statement of value line breaks

### PR DESCRIPTION
**Pull request by sabahirfan**
### JIRA link (if applicable) ###
ROC-2290

### Change description ###
This PR adds line breaks in statement of value in sealed claim form OCCN1. Also added unit test for StatementOfValueProvider.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/StatementOfValueProvider.java [hunk 1]
```diff
@@ -16,73 +17,79 @@
...
+        StringBuilder housingDisprepairContent = new StringBuilder();
+        StringBuilder claimValueContent = new StringBuilder();
 
-        StringBuilder statementOfValue = new StringBuilder();
         final Optional<PersonalInjury> personalInjuryOptional = claim.getClaimData().getPersonalInjury();
         personalInjuryOptional.ifPresent(personalInjury -> {
-            statementOfValue.append(PERSONAL_INJURY);
-            statementOfValue.append(
+            personalInjuryContent.append(PERSONAL_INJURY);
+            personalInjuryContent.append(
```
Comment (timja-kainos) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/StatementOfValueProvider.java]: just do another `.append` don't need `personalInjuryContent`

File: src/main/resources/staff/templates/document/legalSealedClaim.html [hunk 2]
```diff
@@ -298,7 +298,13 @@
     <tr>
       <td><h4>Statement of value</h4></td>
       <td>
-        {{ statementOfValue }}
+        {% if statementOfValue.personalInjury.length > 0 %}
```
Comment (timja-kainos) [inline src/main/resources/staff/templates/document/legalSealedClaim.html]: I think `is defined` is a better check

Comment (sabahirfan) [reply src/main/resources/staff/templates/document/legalSealedClaim.html]: Agree done.

File: src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/StatementOfValueProvider.java (hunk 1 above)
Comment (timja-kainos) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/StatementOfValueProvider.java]: I know that but StringBuilder.append returns a StringBuilder so you can do:
```
personalInjuryContent.append(PERSONAL_INJURY)
  .append(...);
```

Comment (sabahirfan) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/StatementOfValueProvider.java]: Ah. Done.

**Pull request merged**